### PR TITLE
Add support for including agent manager in guest-agent package

### PR DESCRIPTION
**Pull request by ChaitanyaKulkarni28**
/cc @dorileo @drewhli

### CODE REVIEW

File: packaging/googet/google-compute-engine-windows.goospec [hunk 1]
```diff
@@ -36,7 +37,8 @@
   "sources": [{
       "include": [
         "GCEWindowsAgent.exe",
-	"GCEAuthorizedKeysCommand.exe",
+        "GCEWindowsAgentManager.exe",
+	      "GCEAuthorizedKeysCommand.exe",
```
Comment (dorileo) [inline packaging/googet/google-compute-engine-windows.goospec:41]: <tab> vs <space> ?

Comment (ChaitanyaKulkarni28) [reply packaging/googet/google-compute-engine-windows.goospec]: Done, its funny actually, IDE and Github is parsing differently, its looks fine only on either of those and not both,
On IDE it looks like this now :)
Screenshot 2024-10-17 at 4 15 32 PM

**Pull request merged**